docs/application/toc_all.md

```diff
@@ -676,6 +677,7 @@
 #### [Sticker](/application/native/guides/app-management/sticker.md)
 #### [Widget Service](/application/native/guides/app-management/widget-service.md)
 #### [Widget Viewer](/application/native/guides/app-management/widget-viewer.md)
+#### [Cion](/application/native/guides/app-management/cion.md)
```

Comment:
Line 680: Line 81: I think Cion should be placed between [Application Controls] and [Common Application Controls] to maintain alphabetical order.
@mijin85cho please confirm this...

Reply:
it's moved